new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -753,6 +803,111 @@ public static void scaleAndVerifyCluster(String clusterName,
+   *
+   * @param v1pv V1PersistentVolume object to create the persistent volume
+   * @param v1pvc V1PersistentVolumeClaim object to create the persistent volume claim
+   * @param labelSelector String containing the labels the PV is decorated with
+   * @param namespace the namespace in which the persistence volume claim to be created
+   */
+  public static void createPVPVCAndVerify(V1PersistentVolume v1pv,
+                                          V1PersistentVolumeClaim v1pvc,
+                                          String labelSelector,
+                                          String namespace) {
+
+    assertNotNull(v1pv, "v1pv is null");
+    assertNotNull(v1pvc, "v1pvc is null");
+
+    String pvName = v1pv.getMetadata().getName();
+    String pvcName = v1pvc.getMetadata().getName();
+
+    logger.info("Creating persistent volume {0}", pvName);
+    assertTrue(assertDoesNotThrow(() -> createPersistentVolume(v1pv),
+        "Persistent volume creation failed with ApiException "),
+        "PersistentVolume creation failed");
+
+    logger.info("Creating persistent volume claim {0}", pvcName);
+    assertTrue(assertDoesNotThrow(() -> createPersistentVolumeClaim(v1pvc),
+        "Persistent volume claim creation failed with ApiException"),
+        "PersistentVolumeClaim creation failed");
+
+    // check the persistent volume and persistent volume claim exist
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for persistent volume {0} exists "
+                    + "(elapsed time {1}ms, remaining time {2}ms)",
+                pvName,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> pvExists(pvName, labelSelector),
+            String.format("pvExists failed with ApiException when checking pv %s", pvName)));
+
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for persistent volume claim {0} exists in namespace {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                pvcName,
+                namespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> pvcExists(pvcName, namespace),
+            String.format("pvcExists failed with ApiException when checking pvc %s in namespace %s",
+                pvcName, namespace)));
+  }
+
+  /**
+   * Create ConfigMap containing domain scripts.
+   * @param configMapName name of the ConfigMap to create
+   * @param files files to be added in ConfigMap, including domain creation python script and properties files
+   * @param namespace the namespace in which the ConfigMap to be created
+   * @throws IOException when reading files fails
+   */
+  public static void createConfigMapForDomainCreation(String configMapName,
```

Comment:
Since you moved it to utils, the name should be generic. Something like createConfigMapFromFiles(...)

Reply:
fixed